### PR TITLE
feat: Add Docker Hub description for device-uart

### PR DESCRIPTION
**Pull request by vediyappan-villali**
Add service to descriptions/image_descriptions.txt and add MD files to image-overview-templates/new-names

Close #22 in device-uart

## PR Checklist
Please check if your PR fulfills the following requirements:

- [N] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [N] I am not introducing a new dependency (add notes below if you are)
- [N] I have added unit tests for the new feature or bug fix (if not, why?)
- [N] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [N] I have opened a PR for the related docs change (if not, why?)
  N/A

## Testing Instructions
N/A

## New Dependency Instructions (If applicable)
N/A

### CODE REVIEW

**Comment by vediyappan-villali** [comment]
@lenny-intel @cloudxxx8 Please check. Addresses. https://github.com/edgexfoundry/device-uart/issues/22.

File: image-overview-templates/new-names/device-uart-arm64.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+$$./common/quick-reference.md
+
+# Supported tags and respective Dockerfile links
+
+$$./common/version-minnesota.md |
+        - [Dockerfile](https://github.com/edgexfoundry/device-uart/blob/v3.0.0/Dockerfile)
+$$./common/version-levski-2.3.1.md |
+        - [Dockerfile](https://github.com/edgexfoundry/device-uart/blob/v2.3.1/Dockerfile)
+$$./common/version-kamakura.md |
+        - [Dockerfile](https://github.com/edgexfoundry/device-uart/blob/v2.2.0/Dockerfile)
```
Comment (lenny-intel) [inline image-overview-templates/new-names/device-uart-arm64.md:12]: Remove as these do not exists.

File: image-overview-templates/new-names/device-uart.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+$$./common/quick-reference.md
+
+# Supported tags and respective Dockerfile links
+
+$$./common/version-minnesota.md |
+        - [Dockerfile](https://github.com/edgexfoundry/device-uart/blob/v3.0.0/Dockerfile)
+$$./common/version-levski-2.3.1.md |
+        - [Dockerfile](https://github.com/edgexfoundry/device-uart/blob/v2.3.1/Dockerfile)
+$$./common/version-kamakura.md |
+        - [Dockerfile](https://github.com/edgexfoundry/device-uart/blob/v2.2.0/Dockerfile)
```
Comment (lenny-intel) [inline image-overview-templates/new-names/device-uart.md:12]: Remove as these do not exists.

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: image-overview-templates/new-names/device-uart.md (hunk 2 above)
Comment (vediyappan-villali) [reply image-overview-templates/new-names/device-uart.md:12]: Removed.

File: image-overview-templates/new-names/device-uart-arm64.md (hunk 1 above)
Comment (vediyappan-villali) [reply image-overview-templates/new-names/device-uart-arm64.md:12]: Removed.

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**